### PR TITLE
file scan implementation and tests

### PR DESCRIPTION
**Pull request by pangea-andrest**
Test will fail until FileScan is enable on prod

### CODE REVIEW

File: examples/intel/file_scan/file_scan_async.go
Comment (govind-salinas-pangea) [inline examples/intel/file_scan/file_scan_async.go:1]: these files shouldnt't be under examples/intel, they should be under examples/file-scan

File: examples/intel/file_scan/file_scan_async.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/pangea"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/service/file_scan"
+)
+
+const TESTFILE_PATH = "./testdata/testfile.pdf"
+
+func main() {
+	fmt.Println("File Scan async start...")
+	token := os.Getenv("PANGEA_INTEL_TOKEN")
```
Comment (govind-salinas-pangea) [inline examples/intel/file_scan/file_scan_async.go:19]: all the intel names need to be fixed

File: examples/intel/file_scan/file_scan_async.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+	// This should be your own file to scan
+	file, err := os.Open(TESTFILE_PATH)
+	if err != nil {
+		log.Fatal("expected no error got: %v", err)
+	}
+
+	fmt.Println("File Scan request...")
+	_, err = client.Scan(ctx, input, file)
+	if err == nil {
```
Comment (govind-salinas-pangea) [inline examples/intel/file_scan/file_scan_async.go:49]: you cant do this, scan can return a 200

File: examples/intel/file_scan/file_scan_async.go
Comment (pangea-andrest) [reply examples/intel/file_scan/file_scan_async.go:1]: fixed

File: examples/intel/file_scan/file_scan_async.go (hunk 2 above)
Comment (pangea-andrest) [reply examples/intel/file_scan/file_scan_async.go]: fixed

**Review by govind-salinas-pangea — APPROVED** [review]

**Pull request merged**